frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -121,9 +121,14 @@ class UserForm extends Component <ICreateUserFormProps, ICreateUserFormState> {
   };
 
   onIsGlobalUserChange = (value: string): void => {
+    const { formData } = this.state;
     const isGlobalUser = value === UserTeamType.GlobalUser;
     this.setState({
       isGlobalUser,
+      formData: {
+        ...formData,
+        global_role: isGlobalUser ? 'observer' : null,
```

Comment:
Does this prevent setting `global_role` to maintainer or admin?

Reply:
no this just sets the default value back to observer when the user selects a global team radio input. They will still be able to select admin or maintainer.